source/includes/steps-install-mongosync-on-linux.yaml

```diff
@@ -0,0 +1,58 @@
+# This is a parent procedure that provides steps for other procedures to inherit.
+# This procedure is NOT intended for direct inclusion in a tutorial.
+title: Download the tarball.
+stepnum: 1
+level: 4
+ref: download-mongosync-linux
+pre: |
+  After you have installed the required prerequisite packages, download
+  the Cluster-to-Cluster Sync ``.tgz`` tarball from the following link:
+
+  |arrow| |mdb-download-link|
+
+  a. In the :guilabel:`Version` dropdown, select the version of
+     Cluster-to-Cluster Sync to download.
+
+  #. In the :guilabel:`Platform` dropdown, select your operating system
+     version and architecture.
+
+  #. In the :guilabel:`Package` dropdown, select :guilabel:`shell (tgz)`.
+
+  #. Click :guilabel:`Download`.
+---
+title: Extract the files from the downloaded archive.
+stepnum: 2
+level: 4
+ref: extract
+pre: |
+  For example, from a system shell, you can extract using the ``tar`` command:
+
+  .. code-block:: bash
+
+     tar -zxvf mongosync-*.tgz
+---
+title: Ensure the binary is in a directory listed in your ``PATH`` environment variable.
```

Comment:
[question]

Should we include instructions on updating `PATH`?

Reply:
@jmd-mongo They don't on server install docs, not sure whether that's an old standard or current.